src/interface.js

```diff
@@ -44,6 +44,24 @@ class ListTask {
         clearCompleted();
       });
     }
+
+    static initCheckTask = () => {
+      window.addEventListener('load', (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        const check = getChecks();
+        const arr = JSON.parse(localStorage.getItem('list'));
+        for (let i = 0; i < check.length; i += 1) {
+          check[i].addEventListener('change', (e) => {
+            e.preventDefault();
+            if (e.target.checked) {
+              arr[e.target.parentElement.id].completed = true;
+            } else arr[e.target.parentElement.id].completed = false;
+            localStorage.setItem('list', JSON.stringify(arr));
+          });
+        }
+      });
+    }
 }
 
```

Comment:
You did well with your project, kindly note that as part of the requirement, when a user ticks on the todos and clicks `clear all completed` they shouldbe deleted from the list, please kindly implement this feature so youd have a fully functional site 😄 


https://github.com/BiancaMe/ToDoProject/assets/20537712/74ee54bd-7b6d-4151-89f7-04741da179ca